working/1426-extension-types/feature-specification.md

```diff
@@ -458,15 +504,11 @@ etc.*
 Let `E` be an explicit extension type declaration, and consider an
 occurrence of an identifier expression `id` in the body of an instance
 member of `E`. If a lexical lookup of `id` yields a declaration of a
```

Comment:
If we're being persnickety, we probably should say "yields a declaration of an instance member of", since `id` could be a static method of the extension type.

Reply:
Argh, I didn't see this comment when I landed the PR. I'll fix it later.